server/src/pages/shot/view.js

```diff
@@ -343,26 +356,47 @@ class Body extends React.Component {
     let favoriteShotButton = null;
     let trashOrFlagButton = null;
     let editButton = null;
-    const highlight = this.state.highlightEditButton ? <div className="edit-highlight" onClick={ this.onClickEdit.bind(this) } onMouseOver={ this.onMouseOverHighlight.bind(this) } onMouseOut={ this.onMouseOutHighlight.bind(this) }></div> : null;
+    const highlight = this.state.highlightEditButton
+      ? <div className="edit-highlight"
+          onClick={this.onClickEdit.bind(this)}
+          onMouseOver={this.onMouseOverHighlight.bind(this)}
+          onMouseOut={this.onMouseOutHighlight.bind(this)}></div>
+      : null;
     const activeFavClass = this.props.expireTime ? "" : "is-fav";
     const inactive = this.props.isFxaAuthenticated ? "" : "inactive";
 
-    favoriteShotButton = <div className="favorite-shot-button"><Localized id="shotPagefavoriteButton">
-      <button className={`nav-button ${inactive}`} disabled={!this.props.isFxaAuthenticated} onClick={this.onClickFavorite.bind(this)}>
-        <span className={`icon-favorite favorite ${activeFavClass}`} ></span>
-        <Localized id="shotPageFavorite">
-          <span className={`favorite-text favorite ${activeFavClass} `}>Favorite</span>
-        </Localized>
-      </button></Localized></div>;
-
-    const downloadButton = <div className="download-shot-button"><Localized id="shotPageDownloadShot">
-      <button className={`nav-button icon-download`} onClick={this.onClickDownload.bind(this)}
-        title="Download the shot image">
-        <Localized id="shotPageDownload">
-          <span>Download</span>
-        </Localized>
-      </button>
-    </Localized></div>;
+    favoriteShotButton = <div className="favorite-shot-button">
+      <Localized id="shotPagefavoriteButton">
+        <button className={`nav-button ${inactive}`}
+          disabled={!this.props.isFxaAuthenticated}
+          onClick={this.onClickFavorite.bind(this)}>
+          <span className={`icon-favorite favorite ${activeFavClass}`} ></span>
+          <Localized id="shotPageFavorite">
+            <span className={`favorite-text favorite ${activeFavClass} `}>Favorite</span>
+          </Localized>
+        </button></Localized></div>;
+
+    const downloadButton = <div className="download-shot-button">
+      <Localized id="shotPageDownloadShot">
+        <button className="nav-button icon-download" onClick={this.onClickDownload.bind(this)}
+          title="Download the shot image">
+          <Localized id="shotPageDownload">
+            <span>Download</span>
+          </Localized>
+        </button>
+      </Localized></div>;
+
+    const copyButton = <div className="copy-img-button" hidden={!this.state.canCopy}>
+      <Localized id="shotPageCopyButton">
+        <button className="nav-button icon-copy transparent copy"
+          title="Copy image to clipboard"
```

Comment:
Looks like this string wasn't exposed for localization. We'll extract is in #4935